### PR TITLE
[feature1]first feature was implemented

### PR DESCRIPTION
**Pull request by formation2021**
Following changes are made :

- new-feature.txt was added
- one-more-file.txt under feature1 folder

### CODE REVIEW

File: feature1/one-more-file.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+this file was also added in feature1 branch
```
Comment (mouradzoghbi2020) [inline feature1/one-more-file.txt:1]: good to add this line of code

Comment (formation2021) [reply feature1/one-more-file.txt:1]: thanks for your comment mouradzoghbi2020

**Review by mouradzoghbi2020 — APPROVED** [review]
Good Job formation2021

**Review by mouradzoghbi2020 — APPROVED** [review]

**Review by mouradzoghbi2020 — APPROVED** [review]
ok

**Review by mouradzoghbi2020 — APPROVED** [review]

**Pull request merged**